sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/RocksDBStateStoreSuite.scala

```diff
@@ -1834,7 +1834,6 @@ class RocksDBStateStoreSuite extends StateStoreSuiteBase[RocksDBStateStoreProvid
       store = provider.getRocksDBStateStore(2)
       store.createColFamilyIfAbsent(colFamily3, keySchema, valueSchema,
         NoPrefixKeyStateEncoderSpec(keySchema))
-      assert(store.getColumnFamilyId(colFamily3) == 3)
```

Comment:
I'm not sure this test is still valid after you removed the check for column family ID. What's rationale about removing this?

Reply:
I removed this API actually - so removed this part of the test also